docs/community/index.html

```diff
@@ -511,89 +497,27 @@ <h3 id="office-hours">Office Hours</h3>
-</tr>
-<tr>
-<td>February 6, 2019</td>
-<td><a href="https://docs.google.com/document/d/1eg-YjOHaQ7UD28htdNxBR3zufebozXKyI28cl2E11tU/edit">meeting notes</a>, <a href="https://youtu.be/XvpCFaTrtBA">recording</a></td>
-</tr>
-<tr>
-<td>January 30, 2019</td>
-<td><a href="https://docs.google.com/document/d/1eg-YjOHaQ7UD28htdNxBR3zufebozXKyI28cl2E11tU/edit">meeting notes</a>, <a href="https://youtu.be/cTTqIR3muGk">recording</a></td>
-</tr>
-<tr>
-<td>January 23, 2019</td>
-<td><a href="https://docs.google.com/document/d/1eg-YjOHaQ7UD28htdNxBR3zufebozXKyI28cl2E11tU/edit">meeting notes</a>, recording TODO</td>
-</tr>
-<tr>
-<td>January 16, 2019</td>
-<td><a href="https://docs.google.com/document/d/1eg-YjOHaQ7UD28htdNxBR3zufebozXKyI28cl2E11tU/edit">meeting notes</a>, <a href="https://youtu.be/ydA-epcZJQo">recording</a></td>
-</tr>
-<tr>
-<td>January 9, 2019</td>
-<td><a href="https://docs.google.com/document/d/1eg-YjOHaQ7UD28htdNxBR3zufebozXKyI28cl2E11tU/edit">meeting notes</a>, <a href="https://youtu.be/C3zO67lXGrg">recording</a></td>
-</tr>
-<tr>
-<td>January 2, 2020</td>
-<td><a href="https://docs.google.com/document/d/1eg-YjOHaQ7UD28htdNxBR3zufebozXKyI28cl2E11tU/edit">meeting notes</a>, recording didn't work :-( look at the notes</td>
-</tr>
-<tr>
-<td>December 19, 2019</td>
-<td><a href="https://docs.google.com/document/d/1eg-YjOHaQ7UD28htdNxBR3zufebozXKyI28cl2E11tU/edit">meeting notes</a>, <a href="https://youtu.be/FIcySpPkGa4">recording</a></td>
-</tr>
-<tr>
-<td>November, 2019</td>
-<td><a href="https://docs.google.com/document/d/1l_SsVPLMBZ7lm_T4u7ZDBceTTUY71-iEQUPWeOdTAxM/preview">Kubecon 2019 San Diego: API evolution design discussion</a></td>
-</tr>
-<tr>
-<td>November, 2019</td>
-<td><a href="https://docs.google.com/document/d/1AqBaxNX0uS0fb_fSpVL9c8TmaSP7RYkWO8U_SdJH67k/preview">SIG-NETWORK: Ingress Evolution Sync</a></td>
-</tr>
-<tr>
-<td>May, 2019</td>
-<td><a href="https://docs.google.com/document/d/1n8AaDiPXyZHTosm1dscWhzpbcZklP3vd11fA6L6ajlY/preview">Kubecon 2019 Barcelona: SIG-NETWORK discussion (general topics, includes V2)</a></td>
-</tr>
-</tbody>
-</table>
-<h2 id="design-docs">Design docs</h2>
-<table>
-<thead>
-<tr>
-<th>Title</th>
-<th>Description</th>
-</tr>
-</thead>
-<tbody>
-<tr>
-<td><a href="https://docs.google.com/document/d/1BxYbDovMwnEqe8lj8JwHo8YxHAt3oC7ezhlFsG_tyag">API sketch</a></td>
-<td>Sketch of the proposed API</td>
-</tr>
-</tbody>
-</table>
+<p>Meeting agendas and notes are recorded in the <a href="https://docs.google.com/document/d/1eg-YjOHaQ7UD28htdNxBR3zufebozXKyI28cl2E11tU/edit">meeting notes doc</a>.</p>
+<p>All meetings are recorded and automatically uploaded to youtube:
+<a href="https://www.youtube.com/playlist?list=PL69nYSiGNLP2E8vmnqo5MwPOY25sDWIxb">Meeting recordings</a>.</p>
+<p>Some initial recordings of this working group were done manually and can be
+found in the below table:
+| Date               |                                |
```

Comment:
Looks like the table generation got broken here - maybe a line break above this would help?

Reply:
Whoops, fixed. Good catch.